ui/helpers/utils/util.js

```diff
@@ -750,3 +750,16 @@ export const hexToText = (hex) => {
 export const getAvatarFallbackLetter = (subjectName) => {
   return subjectName?.match(/[a-z0-9]/iu)?.[0] ?? '?';
 };
+
+/**
+ * Get checksum address for the address string passed.
+ *
+ * @param {string} address - The address passed.
+ * @returns Checksum address or undefined.
+ */
+export const getChecksumAddress = (address) => {
```

Comment:
We now have `toChecksumAddress`, `toChecksumHexAddress`, and `getChecksumAddress`.

Could we re-use or extend `toChecksumHexAddress`?

Or at least place a new util in the same folder which is `shared/modules/hexstring-utils.ts`?

Reply:
My concern is adding to a generic util that's already almost a thousand lines, as opposed to a targeted file so we can keep our logic modular and organised.

Even if they're different, we now have three very similar functions without obvious differences.

I'd argue that `hextring-utils.ts` is more meaningful than `util.js` since the checksummed address (if successful) is is a hexadecimal string.